<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(162, 128, 137);</string>
  </property>
  <widget class="QPushButton" name="display_but">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>140</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/display.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="general_lab">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;font: 30pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:5px solid black;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>General</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="surgeon_but">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/doctor.png);
border-radius:20px;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="general_but">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>140</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/genral1.png);
border:none;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>general</string>
   </property>
  </widget>
  <widget class="QLabel" name="footpedal_lab">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>610</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;font: 30pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:5px solid black;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>FOOT PEDAL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="display_lab">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>230</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;font: 30pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:5px solid black;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="learner_lab">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;font: 30pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:5px solid black;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Learner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/backarrowimg.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="surgeon_lab">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;font: 30pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:5px solid black;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>SURGEON</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="footpedal_but">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>430</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:transparent;
image: url(:/images/footimg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="learner_but">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>140</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/learner.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
